Assets/MixedRealityToolkit/Extensions/VectorExtensions.cs

```diff
@@ -162,6 +162,56 @@ public static bool IsValidVector(this Vector3 vector)
                    !float.IsInfinity(vector.x) && !float.IsInfinity(vector.y) && !float.IsInfinity(vector.z);
         }
 
+        /// <summary>
+        /// Calculates which side a point in world space is on of a plane
+        /// </summary>
+        /// <param name="initialDirection">The normal direction orthagonal to the plane. </param>
+        /// <param name="initialPosition">The point representing the plane's origin</param>
+        /// <returns>true when the compared point is on the other side of the plane</returns>
+        public static bool IsOtherSideOfPoint(this Vector3 vector, Vector3 initialDirection, Vector3 initialPosition)
+        {
+            Vector3 delta = vector - initialPosition;
+            float dot = Vector3.Dot(delta.normalized, initialDirection);
+
+            return (dot > 0) ? true : false;
```

Comment:
you can just do return dot > 0

Reply:
or return Vector3.Dot(delta.normalized, initialDirection.normalized) > 0